submissionaccessoptions.md

```diff
@@ -0,0 +1,48 @@
+# SubmissionAccessOptions Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/config/submissionaccessoptions**   
+
+Provide access to the existent configurations for the submission access. It returns the list of existent SubmissionAccessConfig.
+The SubmissionAccessConfig aggregates in a single place the configuration needed to the access section.
+
+## Single Submission-Form 
+**/api/config/submissionaccessoptions/<:option-name>**
```

Comment:
After reviewing the PR (https://github.com/DSpace/DSpace/pull/8091) and realizing that this endpoint returns a `AccessConditionConfiguration`, I think we may want to rename the parameter here to be `<:config-name>` or just `<:name>`.   

The term `<:option-name>` may be confusing, as we return a set of `accessConditionOptions` from this endpoint...and we don't want to confuse the name of the AccessConditionOptions with the name of the AccessConditionConfiguration